app/javascript/packages/document-capture/components/submission-complete.jsx

```diff
@@ -31,7 +31,7 @@ function SubmissionComplete({ resource }) {
 
   function handleResponse() {
     if (response.isPending) {
-      if (statusPollInterval !== undefined) {
+      if (Number.isFinite(statusPollInterval)) {
```

Comment:
More context for these changes in the extended commit description of 484665c8b689caf4d5712919e2c4cc5556b41200:

>The `|| undefined` logic will be tripped for 0. The intention was to try to avoid scheduling a poll if the result of "Number" parsing was "NaN", since that would be treated the same as "0" in setTimeout, thus potentially causing a DDOS. Instead, check for finiteness of the value when scheduling.

```js
Number.isFinite(1)
// true
Number.isFinite(0)
// true
Number.isFinite(NaN)
// false
Number.isFinite(undefined)
// false
```